README.md

```diff
@@ -1,23 +1,38 @@
-cdp-portal-perf-tests
+# cdp-portal-perf-tests
 
+* [Development](#development)
+  * [Run localStack](#run-localstack)
+  * [Create bucket](#create-bucket)
+  * [Build](#build)
+  * [Run](#run)
+* [Licence](#licence)
+  * [About the licence](#about-the-licence)
 
-- [Licence](#licence)
-  - [About the licence](#about-the-licence)
+## Development
 
+### Run localStack
 
+```bash
+docker run --pull=always -d -p 4566:4566 -p 4510-4559:4510-4559 localstack/localstack:latest
+```
 
-## Build
+### Create bucket
 
-```
-docker build -t defra/cdp-portal-perftest .
+```bash
+awslocal s3 mb s3://cdp-perf-test-test-results --endpoint-url http://localhost:4566
 ```
 
-## Run
+### Build
 
 ```
-docker run -it --rm -v $PWD/reports:/opt/perftest/reports defra/cdp-portal-perftest
+docker build --no-cache -t defradigital/cdp-portal-perf-tests .
 ```
 
+### Run
+
+```
+docker run -e S3_ENDPOINT='http://host.docker.internal:4566' -e RESULTS_OUTPUT_S3_PATH='s3://cdp-perf-test-test-results/cdp-portal-perf-tests/a845a2f1-397d-439e-9ea0-e62bb7377e07' -e AWS_ACCESS_KEY_ID='test' -e AWS_SECRET_ACCESS_KEY='test' -e AWS_SECRET_KEY='test' -e AWS_REGION='eu-west-2' -e ENVIRONMENT='perf-test' defradigital/cdp-portal-perf-tests
+```
```

Comment:
We should perhaps create a compose file here with these as env vars.?

Reply:
yeah sounds good. I also want to create a compose with localstack and buckets in the PFE